src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -612,8 +615,57 @@ public virtual ProtoCore.Type GetTypeHintForOutput(int index)
         {
              return ProtoCore.TypeSystem.BuildPrimitiveTypeObject(ProtoCore.PrimitiveType.kTypeVar);
         }
-        #endregion
 
+      
+        /// <summary>
+        /// Gets or sets a value indicating whether the node is frozen.
```

Comment:
We should update the comments:
> A flag indicating whether the node is frozen.
> When a node is frozen, the node, and all nodes downstream will not participate in execution.
> This will return true if any upstream node is frozen or if the node was explicitly frozen.